### ISSUE TITLE
Checkboxes with multiline label overlap each other

### ISSUE THREAD

**Issue by Stefei** [siemens/ix#1226]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [X] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### What happened?

Stacking multiple checkboxes with multiline labels results in them overlapping each other:

![grafik](https://github.com/siemens/ix/assets/82656629/cdf70e07-6d88-4dd5-a1d8-d669a371334a)

### What type of frontend framework are you seeing the problem on?

Angular, JavaScript

### Which version of iX do you use?

v2.1.0

### Code to produce this issue.

```shell
<div style="display: flex; flex-direction: column">
    <input type="checkbox" id="checkbox_01" />
      <label for="checkbox_01">Long Label 1: Lorem ipsum dolor sit amet, consetetur sadipscing elitr</label>
      <input type="checkbox" id="checkbox_02" />
      <label for="checkbox_02">Long Label 2: Lorem ipsum dolor sit amet, consetetur sadipscing elitr</label>
      <input type="checkbox" id="checkbox_03" />
      <label for="checkbox_03">Long Label 3: Lorem ipsum dolor sit amet, consetetur sadipscing elitr</label>
    </div>
```

**Comment by ridvandmrc** [comment]
Hi @Stefei ,
You can manipulate it your side like:
```html
<div style="display: flex; flex-direction: column">
  <input type="checkbox" id="checkbox_01" />
    <label style="display:block; white-space: nowrap;overflow: hidden;text-overflow: ellipsis;max-width: 90%;" for="checkbox_01">Long Label 1: Lorem ipsum dolor sit amet, consetetur sadipscing elitr</label>
    <input type="checkbox" id="checkbox_02" />
    <label style="display:block; white-space: nowrap;overflow: hidden;text-overflow: ellipsis;max-width: 90%;"  for="checkbox_02">Long Label 2: Lorem ipsum dolor sit amet, consetetur sadipscing elitr</label>
    <input type="checkbox" id="checkbox_03" />
    <label style="display:block; white-space: nowrap;overflow: hidden;text-overflow: ellipsis;max-width: 90%;"  for="checkbox_03">Long Label 3: Lorem ipsum dolor sit amet, consetetur sadipscing elitr</label>
  </div>
```

result is like:
![image](https://github.com/siemens/ix/assets/15720132/a56bffa5-770f-4aae-a656-c42b27b56b4f)


Other way is arrange the height of **label**:
```html
  <div style="display: flex; flex-direction: column;margin-top: 4rem;">
    <input type="checkbox" id="checkbox_01" />
      <label style="height:unset; max-height: unset;" for="checkbox_01">Long Label 1: Lorem ipsum dolor sit amet, consetetur sadipscing elitr</label>
      <input type="checkbox" id="checkbox_02" />
      <label style="height:unset; max-height: unset;" for="checkbox_02">Long Label 2: Lorem ipsum dolor sit amet, consetetur sadipscing elitr</label>
      <input type="checkbox" id="checkbox_03" />
      <label style="height:unset; max-height: unset;" for="checkbox_03">Long Label 3: Lorem ipsum dolor sit amet, consetetur sadipscing elitr</label>
    </div>
```

result is like:

![image](https://github.com/siemens/ix/assets/15720132/e6e567c6-2c0b-4602-96f3-f1092395c772)


it is up to you

**Comment by nuke-ellington** [comment]
Will be closed by #1273
